signac_dashboard/modules/file_opener.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2019 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import os
+
+from flask import abort, render_template, request
+from jinja2.exceptions import TemplateNotFound
+
+from signac_dashboard.module import Module
+
+
+class FileOpener(Module):
```

Comment:
We might be able to make an equivalent of the FileList module that opens instead of downloading the file. Or both, with buttons for either open/download action. Perhaps with a regex filter of filenames to display?